instance/migrations/0078_field_defaults.py

```diff
@@ -85,7 +85,7 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='openedxinstance',
             name='mongodb_server',
-            field=models.ForeignKey(blank=True, default=instance.models.mixins.database.select_random_mongodb_server, null=True, on_delete=django.db.models.deletion.PROTECT, to='instance.MongoDBServer'),
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.PROTECT, to='instance.MongoDBServer'),
```

Comment:
@rocioar I think modifying old migrations is not a good idea. Maybe you could preserve the `select_random_mongodb_server` method where it was? Or maybe just move it to some "legacy" package.

Alternative approach (probably better, but more work) is to squash the migrations. Looks like there are over a hundred mmigrations now, so I bet we'll never need anything below 0090. I think Django provides some means for doing so.

@pomegranited @bradenmacdonald Might be good to know your thoughts about squashing migrations.

Reply:
@e-kolpakov @rocioar Squashing migrations sounds like a good idea, yep.

Reply:
Squashed migrations from 0090 down, I kept the files, so that deployment can go well, and once deployment is done, we can remove them :)